src/Okta.Sdk.IntegrationTests/PoliciesScenarios.cs

```diff
@@ -432,6 +462,158 @@ public async Task CreatePasswordPolicyRule()
             }
         }
 
+        [Fact]
+        public async Task CreateAccessPolicyPolicyRule()
+        {
+            var client = TestClient.Create();
+            var guid = Guid.NewGuid();
+
+            var createdApp = await client.Applications.CreateApplicationAsync(new CreateOpenIdConnectApplication
+            {
+                Label = $"dotnet-sdk: AddOpenIdConnectApp {guid}",
+                ClientUri = "https://example.com/client",
+                LogoUri = "https://example.com/assets/images/logo-new.png",
+                ResponseTypes = new List<OAuthResponseType>
+                {
+                    OAuthResponseType.Token,
+                    OAuthResponseType.IdToken,
+                    OAuthResponseType.Code,
+                },
+                RedirectUris = new List<string>
+                {
+                        "https://example.com/oauth2/callback",
+                        "myapp://callback",
+                },
+                PostLogoutRedirectUris = new List<string>
+                {
+                    "https://example.com/postlogout",
+                    "myapp://postlogoutcallback",
+                },
+                GrantTypes = new List<OAuthGrantType>
+                {
+                    OAuthGrantType.Implicit,
+                    OAuthGrantType.AuthorizationCode,
+                },
+                ApplicationType = OpenIdConnectApplicationType.Web,
+            });
+
+            var accessPolicyId = createdApp.GetAccessPolicyId();
```

Comment:
How is this id fetched from create app response? I see this interface defined in `IApplication`, but do not see the implementation. Also, I do not see a sample response that contains this Id in documentation at https://developer.okta.com/docs/reference/api/apps/#response-example-self-service-application-assignment-not-available.

Reply:
Got answered offline.